lib/nerves/release.ex

```diff
@@ -167,4 +169,58 @@ defmodule Nerves.Release do
       {to_string(app), to_string(opts[:vsn]), Path.expand(opts[:path] || "")}
     end
   end
+
+  defp check_vm_args_compatibility!(release) do
+    Mix.shell().info([:yellow, "* [Nerves] ", :reset, "validating vm.args"])
+    vm_args_path = Mix.Release.rel_templates_path(release, "vm.args.eex")
+
+    if not File.exists?(vm_args_path) do
+      Mix.raise("Missing required #{vm_args_path}")
+    end
+
+    if Version.match?(System.version(), ">= 1.15.0") do
+      File.stream!(vm_args_path, [], :line)
+      |> Stream.with_index(1)
+      |> Enum.filter(fn {line, _} -> String.contains?(line, "Elixir.IEx.CLI") end)
+      |> case do
+        [] ->
+          :ok
+
+        lines ->
+          formatted =
+            for {line, num} <- lines do
+              [
+                "* ",
+                vm_args_path,
+                ":",
+                to_string(num),
+                "\n",
+                :red,
+                "\n  - ",
+                line,
+                :green,
+                "  + the_new_way\n",
```

Comment:
Prob shouldn't merge until we have this new way figured out

Reply:
The new way is to replace this:

```
-noshell
-user Elixir.IEx.CLI
```

with this:

```
-noshell
-user elixir
-run elixir start_iex
```